src/explore-education-statistics-admin/src/pages/release/datablocks/components/chart/ChartReferenceLinesConfiguration.tsx

```diff
@@ -134,7 +136,34 @@ export default function ChartReferenceLinesConfiguration({
             }}
             validationSchema={Yup.object<AddFormValues>({
               label: Yup.string().required('Enter label'),
-              position: Yup.string().required('Enter position'),
+              otherAxisPosition: Yup.number().test({
+                name: 'otherAxisPosition',
+                message: 'Enter a valid position',
+                test: (value: number) => {
+                  if (!value) {
+                    return true;
+                  }
+                  if (type === 'minor') {
+                    return value >= 0 && value <= 100;
+                  }
+                  return minorAxisDomain
+                    ? value >= minorAxisDomain?.min &&
+                        value <= minorAxisDomain.max
+                    : true;
+                },
+              }),
+              position: Yup.string()
+                .required('Enter position')
+                .test({
+                  name: 'axisPosition',
+                  message: 'Enter a valid position',
```

Comment:
Same as the comment on the `otherPosition` validation, we could use a clearer validation message here